### ISSUE TITLE
Confused about Boogie releases

### ISSUE THREAD

**Issue by zvonimir** [boogie-org/boogie#378]
I don't quite get how the timing works. We merged my pull request into master branch yesterday, but the nuget package is still 6 days old. Is that expected? I thought that nuget gets created with every push to master branch.

I am also confused about the github releases. See here: https://github.com/boogie-org/boogie/releases/tag/v2.8.22
The latest release there is even older than what is on nuget. Again, my understanding was that both nuget and github releases are done at the same time.

Could someone please clarify this for me? What am I missing here? Thanks!

**Comment by bkragl** [comment]
Because GitVersionTask caused more pain than joy and creating a release with every push to master seemed noisy, we switched back to a more manual release process when CI was moved from Travis to GitHub Actions (see #332). The steps to create a release are described in the readme under "Versioning and Release".

The nuget package is created and released by the CI system. We should add a step that also marks releases on GitHub.
